src/main/java/se/sundsvall/precheck/api/model/Status.java

```diff
@@ -0,0 +1,17 @@
+package se.sundsvall.precheck.api.model;
+
+public enum Status {
+
+    ACTIVE("ACTIVE"),
```

Comment:
What is the purpose with the private member here?

Reply:
For the moment, the 'ACTIVE' constant in the enum is being used at multiple points in the code when querying the 'Party-Assets' integration. It is specified so that the integration does not provide permits for statuses that are either 'EXPIRED' or 'BLOCKED'.  This enum is also a copy of the Status of the 'Party-Assets' integration's Status enum.

Or am I misunderstanding the question? If so, please, I would be delighted by even a small explanation of what the potential code misuse is here!

Reply:
No worries, I rephrase myself:

Change this class to:
```
public enum Status {
    ACTIVE,
    EXPIRED,
    BLOCKED;
}
```